std/range/package.d

```diff
@@ -1415,7 +1415,7 @@ private struct ChooseResult(R1, R2)
     private bool r1Chosen;
 
     private static auto ref actOnChosen(alias foo, ExtraArgs ...)(ref ChooseResult r,
-            auto ref ExtraArgs extraArgs)
+            auto ref ExtraArgs extraArgs) @trusted
```

Comment:
We can't just set a function that takes an arbitrary function as `@trusted`

Reply:
Yes, I'm aware; I'm thinking about a way how to cope with that